src/registry/ecr.go

```diff
@@ -121,28 +126,50 @@ func (r *RegistryScan) GetScanFindings(ctx context.Context, digestInfo RegistryI
 		},
 	})
 
-	var out *ecr.DescribeImageScanFindingsOutput
+	findings := []types.ImageScanFinding{}
+	enhancedFindings := []types.EnhancedImageScanFinding{}
+
+	imageScanFindings := types.ImageScanFindings{}
 
 	for pg.HasMorePages() {
```

Comment:
fingers crossed rsc and the gang get those custom iterators & range over function stuff done one day!